gobblin-example/src/main/java/gobblin/example/wikipedia/WikipediaExtractor.java

```diff
@@ -140,6 +144,24 @@ public WikipediaExtractor(WorkUnitState workUnitState) throws IOException {
     this.reader = new WikiResponseReader();
   }
 
+  private String readProp(String key, WorkUnitState workUnitState) {
```

Comment:
I'm fine with leaving this in, but concerned that this is a hack that hides a more serious problem with workunit state and job state ambiguity.